mmv1/third_party/terraform/services/vertexai/iam_vertex_endpoint_test.go.erb

```diff
@@ -0,0 +1,364 @@
+<% autogen_exception -%>
+package vertexai_test
+
+<% unless version == 'ga' -%>
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccVertexAIEndpointIamBindingGenerated(t *testing.T) {
```

Comment:
```suggestion
func TestAccVertexAIEndpointIamBinding(t *testing.T) {
```

This test can be renamed since it is not being generated by building mmv1.